### ISSUE TITLE
Jetelements - Pricing table dynamic user role feature.

### ISSUE THREAD

**Issue by josiase95** [CrocoBlock/suggestions#4333]
Hello CB,

I think it would be a game changer to the pricing table widget, if you would be able to set user role dynamic conditions to one, and style if after that. So lets say a user has already purchased a premium subscription of 5 euro/month, but wants to upgrade or downgrade from that. If that person goes to the pricing table page, the pricing table of  the current subscription will have a grey overlay and the button will say "already bought" or whatever u make.

So in short. Make it possible to style the pricing table by user role conditions. :-)

**Comment by aalexandr21** [comment]
Hi 

You can achieve this using the Dynamic Visibility from JetEngine https://crocoblock.com/knowledge-base/features/dynamic-visibility-overview/ 

Add and style the widget for users with role A, and show this widget only for users with role A
